packages/mail/src/templates/experiment-remove-wrong-meter.tsx

```diff
@@ -0,0 +1,46 @@
+import { Container, Head, Html, Preview, Text } from "@react-email/components";
+import { Footer } from "../components/footer";
+import { Header } from "../components/header";
+import { Main } from "../components/main";
+
+interface Props {
+    name: string;
+}
+
+export default function ExperimentRemovedWrongMeterTemplate({ name }: Props) {
+    return (
+        <Html lang="de">
+            <Preview>Teilnahme nicht möglich.</Preview>
+            <Head />
+            <Main>
+                <Header>Teilnahme nicht möglich</Header>
+                <Container className="px-4">
+                    <Text>
+                        Hallo {name}, <br />
+                        <br />
+                        wir danken Ihnen sehr herzlich für Ihr Interesse an unserem Projekt und Ihre Bereitschaft zur
+                        Teilnahme. Leider sind unsere Sensoren mit Ihrem Zählermodell nicht kompatibel. Sollten Sie
+                        dennoch am Projekt teilnehmen wollen, melden Sie sich bitte nochmals bei uns. Es ist unter
+                        Umständen möglich mit Ihrem Messstellenbetreiber (der Firma, die Ihren Zähler managt) über einen
```

Comment:
"Ich glaube, "managt" versteht nicht unbedingt jeder
```suggestion
                        Umständen möglich mit Ihrem Messstellenbetreiber (der Firma, die Ihren Zähler verwaltet) über einen
```

Reply:
Sagen Simon und ich auch 👯
Vorgabe kam von Jessy 😅

Reply:
Da haben wir auch schon (also Malte und ich) drüber diskutiert (die Deutsche-Variante ist so komisch).
Musst du aber wohl mit @jessy2t abklären, die hat den Text, wenn ich richtig verstanden habe, verfasst.